gpsnavi: Fix compilation with OpenSSL 1.1

Handle the ERR_load_crypto_strings and ERR_free_strings functions no
longer being present in OpenSSL 1.1.

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/src/sms/sms-core/SMCAL/SMCAL.c b/src/sms/sms-core/SMCAL/SMCAL.c
index eadab8f..2dfe1f3 100755
--- a/src/sms/sms-core/SMCAL/SMCAL.c
+++ b/src/sms/sms-core/SMCAL/SMCAL.c
@@ -158,7 +158,9 @@ void SC_CAL_Initialize_OpenSSL() {
 	// 初期化
 	ERR_load_BIO_strings();
 	SSL_load_error_strings();
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
 	ERR_load_crypto_strings();
+#endif
 	OpenSSL_add_all_algorithms();
 
 	// SSLの初期化(戻り値は常に1)
@@ -542,7 +544,9 @@ E_SC_CAL_RESULT SC_CAL_DisConnect(SMCAL *cal)
 			if (NULL != cal->ssl.ctx) {
 				SSL_CTX_free((SSL_CTX*)cal->ssl.ctx);
 			}
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
 			ERR_free_strings();
+#endif
 
 			cal->ssl.ssl = NULL;
 			cal->ssl.ctx = NULL;
